### PR TITLE
Allow activity packages to supply their own files

### PR DESCRIPTION
**Pull request by houshuang**
Inspired by Jenny's recent PR, which refers to several files in example data, which are of course not available outside of her local computer, I made a change where activity types can provide their own files, in the directory:

`/ac/ac-ranking/clientFiles/anything.ext`

and then refer to these through the URL:

`/file?name=ac/ac-ranking/anything.ext`

This will mainly be useful for exampleData which requires files. As an example, I changed ac-ck-board to load it's image locally, instead of from imgur.

### CODE REVIEW

**Comment by jkolsen** [comment]
This looks fine to me if it looks good to you, Louis. When testing it, I could not tell, does this change the user interaction at all or do we just put our files in that folder?

**Comment by lfaucon** [comment]
@jkolsen It's not a feature for user, but for developers building activity packages. Can you try to fix the missing exampleData files for ac-ranking on top of this PR?

**Comment by jkolsen** [comment]
Done. The new ranking files appear to work.

File: ac/ac-ranking/src/meta.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 const trainConfig = {
   title: 'Which interface is the best?',
   guidelines:
-    '<p>Please drag the items in order.</p><p><img src="/file?name=cjg7q0o6400063i6ivwiot4op" style={{padding: "5" }} width="306" height="146"/><img src="/file?name=cjg7q4md100073i6impu8qvo4" border="2" width="306" height="112"/><img src="/file?name=cjg7q50t600083i6i0spm9jdi" border="2" width="300" height="262"/><img src="/file?name=cjg7q58xy00093i6icte72u16" border="2" width="302" height="98"/></p>',
+    '<p>Please drag the items in order.</p><p><img src="/file?name=ac/ac-ranking/form.png" style={{padding: "5" }} width="306" height="146"/><img src="/file?name=/file?name=ac/ac-ranking/graphical.png" border="2" width="306" height="112"/><img src="/file?name=ac/ac-ranking/dragndrop.png" border="2" width="300" height="262"/><img src="/file?name=ac/ac-ranking/command.png" border="2" width="302" height="98"/></p>',
```
Comment (lfaucon) [inline ac/ac-ranking/src/meta.js]: `/file?name=/file?name=` there was a typo for the graphical image

**Review by lfaucon — APPROVED** [review]

**Pull request merged**